src/config/chainEndpoints.ts

```diff
@@ -15,6 +16,15 @@ export const providerEndpoints: ChainProvider[] = [
     endpoint: 'wss://rpc.plasmnet.io/',
     favicon: 'favicon.png',
   },
+  {
+    networkAlias: 'shiden-shell',
+    displayName: 'Shiden Network (Kusama)',
+    info:
+      'Smart contract platform for decentralized applications (dapps) on the Kusama network',
+    endpoint: 'wss://rpc.shiden.plasmnet.io',
```

Comment:
Shiden network uses the `plasmCollatorDefinitions` for the type registry (the export name will change soon) https://github.com/PlasmNetwork/api/blob/2d52d49d8e95dd067c289578593ab340fbcda4b4/packages/types/src/networkSpecs/plasmCollator.ts#L3

Can you fix [this line](https://github.com/PlasmNetwork/plasm-apps-vue/blob/1209def72e639ffa03174ae21dbd52771e39cebb/src/api/polkadot/connectApi.ts#L61) to make it so that different networks can import from different type defs?